composer/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "composer",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/node": "^18.8.4",
+    "@types/yargs": "^17.0.13",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
+  },
+  "dependencies": {
+    "@iarna/toml": "^2.2.5",
+    "@types/iarna__toml": "^2.0.2",
+    "sqlite": "^4.1.2",
+    "sqlite3": "^5.1.2",
+    "yargs": "^17.6.2"
+  }
+}
```

Comment:
Thanks for bringing this, yeah I considered but as it wasn't present by default on `ubuntu-latest`  and we weren't using container images on the workflow I didn't want to introduce it. But if we can I am happy to use it if everyone agrees.

Reply:
Installing it seems to be easy enough: https://github.com/denoland/setup-deno

I think this would be a good usecase for it, should make this script a lot more maintainable.